src/main.c

```diff
@@ -148,6 +157,8 @@ int main(int argc, char *argv[])
     char pql_conf_buf[LCORE_CONF_BUFFER_LEN];
     int pql_conf_buf_len = LCORE_CONF_BUFFER_LEN;
 
+    enable_coredump();
+
```

Comment:
It's advised to make some improvements for this feature.
1. Coredump feature needs to be enabled/disabled from user config files or command option.
2. Coredump soft/hard resource limits should be set by users.